core/src/banking_stage.rs

```diff
@@ -120,6 +120,8 @@ pub struct ExecuteAndCommitTransactionsOutput {
     // how many such transactions were committed
     commit_transactions_result: Result<(), PohRecorderError>,
     execute_and_commit_timings: LeaderExecuteAndCommitTimings,
+    // True if transaction was-executed()
+    transactions_execute_and_record_status: Vec<bool>,
```

Comment:
I think it's better to remove this field and move this info into `commit_transactions_result` so that its type becomes `Result<Vec<bool>, PohRecorderError>`